apps-rendering/src/server/aws.ts

```diff
@@ -1,18 +1,14 @@
-import SSM from 'aws-sdk/clients/ssm';
-import type { Credentials } from 'aws-sdk/lib/core';
-import {
-	CredentialProviderChain,
-	EC2MetadataCredentials,
-	SharedIniFileCredentials,
-} from 'aws-sdk/lib/core';
+import { fromIni, fromInstanceMetadata } from '@aws-sdk/credential-providers';
+import { chain as providerChain } from '@smithy/property-provider';
+import { SSM } from '@aws-sdk/client-ssm';
 import { Region } from './appIdentity';
 
-const credentialProvider = new CredentialProviderChain([
-	(): Credentials => new SharedIniFileCredentials({ profile: 'mobile' }),
-	(): Credentials => new EC2MetadataCredentials(),
+const credentialProvider = providerChain([
+	fromIni({ profile: 'mobile' }),
+	fromInstanceMetadata(),
```

Comment:
The types fail here, do you have any idea of how to handle this in the new API?

```
Argument of type 'AwsCredentialIdentityProvider[]' is not assignable to parameter of type 'Provider<unknown>'.
```

I’ve tried suppliying `AwsCredentialIdentity` to the `providerChain` generic to no avail

Reply:
providerChain accepts list of providers, and not an array.

Fixed in https://github.com/guardian/dotcom-rendering/pull/9411/commits/6fce472306a63cc2889886a71923c4e25f2a47d5

Reply:
🤦‍♂️ once you see it…

The TS error wasn't too helpful, though, it's east to miss!